alcs-api/src/providers/typeorm/migrations/1657753458415-healthcheck.ts

```diff
@@ -1,15 +0,0 @@
-import { MigrationInterface, QueryRunner } from 'typeorm';
```

Comment:
I assume since we have enough examples of migrations this one will be included to the initial migration, correct?

Reply:
Yes, we will have to remember to add this back into the initial migration when it gets setup